.github/workflows/e2e.yaml

```diff
@@ -17,7 +17,8 @@ jobs:
       - uses: jumpstarter-dev/jumpstarter-e2e@main
```

Comment:
I tried to replace main with " ${{ github.event.pull_request.base.ref }}" but it seems like github doesn't like it... we can change to release-0.6 e2e branch later on the release branch